src/pages/home/report/ReportActionsView.js

```diff
@@ -365,7 +344,6 @@ class ReportActionsView extends React.Component {
         }
 
         this.setUpUnreadActionIndicator();
```

Comment:
I think this also can be performed when the component mounts now ? 🤔 

And we should have no need for something like 

`this.shouldShowUnreadActionIndicator`

Reply:
Now that you brought it up I can see. I just focused on the logic around report loading, there are probably other things that can be updated too